core/commands/copy.py

```diff
@@ -215,11 +215,22 @@ def validate_args(self):
                 raise OSCError('OUTFILE_DIR_NOT_DIR',
                                {'dir': self.args.outfile_dir})
 
-        # Ensure all the given ddl files are readable
+        # Ensure all the given ddl files are readable and
+        # can be decoded with the current charset
         for filepath in self.args.ddl_file_list:
             if not util.is_file_readable(filepath):
                 raise OSCError('FAILED_TO_READ_DDL_FILE',
                                {'filepath': filepath})
+            charset = getattr(self.args, 'charset')
```

Comment:
How about making `ascii` a default value for option "--charset" so that we can remove this `if` block.